crates/sui-types/src/move_package.rs

```diff
@@ -63,6 +64,10 @@ impl MovePackage {
     pub fn disassemble(&self) -> SuiResult<BTreeMap<String, Value>> {
         disassemble_modules(self.module_map.values())
     }
+
+    pub fn normalize(&self) -> SuiResult<BTreeMap<String, Module>> {
```

Comment:
I'm a bit worried about the creep of APIs like this attached `MovePackage`. There is no guarantee that a `MovePackage` value deserializes or passes the verifier, since the type is also used for failed transactions. (the one below is fine)